packages/account-v2/src/containers/PersonalDetailsForm/validation.ts

```diff
@@ -0,0 +1,151 @@
+import * as Yup from 'yup';
+import { GetSettings, ResidenceList, StatesList } from '@deriv/api-types';
+
+type initialValues = {
```

Comment:
You can infer the type of the form data from Yup schema itself
`Yup.InferType<ReturnType<typeof schema>>;`

Reply:
Not everything will be validate inside the schema, for example, taxIdentificationNumber is not being validate so typescript will complain that taxIdentificationNumber is undefined. How can we improve the typing for this 🙏